lib/src/room.dart

```diff
@@ -618,6 +618,14 @@ class KurentoRoom extends ChangeNotifier implements Room {
           // Agent-initiated end message, but the impact of that is low (the Explorer can end the call themselves).
           _getServiceRequestStatusTimer?.cancel();
 
+          final bool hasOutgoingVideoSenderTrack = _connectionByTrackId[_localTrackId!]?.ownsVideoTrack != true;
+          if (!_hasVideoTrack || !hasOutgoingVideoSenderTrack) {
```

Comment:
Maybe add and log `_isVideoMuted` to this list? If it happens to be `true`, we'll start the call without video too.